langchain/src/base_language/index.ts

```diff
@@ -0,0 +1,40 @@
+import { BasePromptValue, LLMResult } from "../schema/index.js";
```

Comment:
I had to move this out to avoid circular dependency: BaseLanguageModel in langchain/schema used CallbackManager, CallbackManager used LLMResult and other things in langchain/schema